test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -234,6 +247,72 @@ class AccountListPage {
     }
   }
 
+  async addAccount(accountType: ACCOUNT_TYPE, accountName: string) {
+    await this.driver.clickElement(this.createAccountButton);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let addAccountButton: any;
+    switch (accountType) {
+      case ACCOUNT_TYPE.ETHEREUM:
+        addAccountButton = this.addEthereumAccountButton;
+        // await this.driver.clickElement(this.addEthereumAccountButton);
+
+        break;
+      case ACCOUNT_TYPE.BITCOIN:
+        addAccountButton = this.addBtcAccountButton;
+        // await this.driver.clickElement(this.addBtcAccountButton);
+        break;
+      case ACCOUNT_TYPE.SOLANA:
+        addAccountButton = this.addSolanaAccountButton;
+        // await this.driver.clickElement(this.addSolanaAccountButton);
+        break;
+      default:
+        console.log('Account type does not match with any of the options');
+        return;
```

Comment:
This should throw to be future proof, otherwise this would fail silently and the tests depending on this function will most likely fail (which would make "debugging" slightly harder IMO).

```suggestion
        throw new Error('Account type not supported');
```